src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs

```diff
@@ -42,4 +42,26 @@ public static void Property_KeyEntropyMode()
         TransportSecurityBindingElement securityBindingElement = new TransportSecurityBindingElement();
         Assert.Equal(securityBindingElement.KeyEntropyMode, SecurityKeyEntropyMode.CombinedEntropy);
     }
+
+    [WcfFact]
+    public static void Method_CreateIssuedTokenOverTransportBindingElement()
+    {
+        IssuedSecurityTokenParameters tokenParameters = new IssuedSecurityTokenParameters();
+        TransportSecurityBindingElement securityBindingElement = SecurityBindingElement.CreateIssuedTokenOverTransportBindingElement(tokenParameters);
+
+        Assert.Contains(tokenParameters, securityBindingElement.EndpointSupportingTokenParameters.Endorsing);
+        Assert.Equal(System.ServiceModel.MessageSecurityVersion.Default, securityBindingElement.MessageSecurityVersion);
```

Comment:
Please do a one time manual verification that MessageSecurityVersion.Default is the same in Core as it is in Full Framework.

Reply:
Verified, it's the same as in Full Framework.